Tools/autotest/quadplane.py

```diff
@@ -164,12 +168,17 @@ def test_airmode(self):
             self.progress("Waiting for Motor1 to (not) speed up")
             self.delay_sim_time(spool_delay)
             self.wait_servo_channel_value(5, spin_arm_pwm, comparator=operator.le)
+            self.wait_servo_channel_value(6, spin_arm_pwm, comparator=operator.le)
+            self.wait_servo_channel_value(7, spin_arm_pwm, comparator=operator.le)
+            self.wait_servo_channel_value(8, spin_arm_pwm, comparator=operator.le)
 
             self.progress("disarming with switch")
             self.disarm_motors_with_switch(arm_ch)
             self.progress("Waiting for Motor1 to stop")
             self.wait_servo_channel_value(5, min_pwm, comparator=operator.le)
             self.wait_ready_to_arm()
+        # remove attitude error
+        self.set_parameter("AHRS_TRIM_X", 60*math.pi/180)
```

Comment:
Probably better to revert this to the previous value - so take a backup and restore it here.  You could use a context instead - but probably overkill.